src/components/footer/ConnectedFooter.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react-redux'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->